CHANGELOG.md

```diff
@@ -28,6 +28,126 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v7.0.0-beta5](https://github.com/shopsys/shopsys/compare/v7p.0.0-beta4...v7.0.0-beta5) - 2019-01-17
```

Comment:
typo `v7p.0.0-beta4`

Reply:
also there is yesterday's date (2019-01-17)

Reply:
The date will be fixed by releaser in release stage

Reply:
OK then.